### PR TITLE
Add bindings for CDPSession and Target

### LINKED ISSUE #39: Bindings for CDPSession
https://github.com/GoogleChrome/puppeteer/blob/v1.2.0/docs/api.md#class-cdpsession

### PR DESCRIPTION
**Pull request by jihchi**
Add bindings:
```
  Page
    ✓ target()
  Target
    ✓ page (1ms)
    ✓ type
    ✓ url (1ms)
    ✓ createCDPSession (1ms)
  CDPSession
    ✓ detach (190ms)
    ✓ send (49ms)
```

Fix #39 and #29

### CODE REVIEW

**Comment by jihchi** [comment]
No problem! Fixed in commit 08d613d

File: src/CDPSession.re [hunk 1]
```diff
@@ -0,0 +1,6 @@
+type t;
+
+[@bs.send] external detach : t => Js.Promise.t(unit) = "";
+
+[@bs.send]
+external send : (t, string, Js.t({..})) => Js.Promise.t(Js.t({..})) = "";
```
Comment (zploskey) [inline src/CDPSession.re]: The third argument `params` is optional.

Comment (jihchi) [reply src/CDPSession.re]: Fixed in commit a980bc6

**Comment by jihchi** [comment]
@zploskey PR is done, could you please review it?

**Pull request merged**